### DISCUSSION TITLE
PDF Viewer - 19.7.1 - After Angular Upgrade not working

### DISCUSSION THREAD

**Discussion by manzoorsharif123** [stephanrauh/ngx-extended-pdf-viewer#2324] [Q&A]
Hi Community, After updating Angular version from 6.3 to 17, PDF viewer is not not working and not showing any content in UAT environment.
In dev / local it is working perfectly fine.

![image](https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/108550245/a98a80a3-39a9-464a-bd0e-fefd9642cea6)
![image](https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/108550245/4ba4dc43-e1f4-4bd2-a032-df72d3079c1e)

**Comment by stephanrauh** [comment]
You need to configure your server such that it detects *.mjs files as JavaScript files. I've described this for several servers here: https://pdfviewer.net/extended-pdf-viewer/troubleshooting

I'm not sure why my library switches to ES5 files. Maybe that's because I've raised the minimum required browser version, and your browser is older. Or maybe updating to version 20.0.1 helps. One of the versions I've published didn't detect the correct browser version correctly. However, as far as I remember, I quickly uploaded a patch version fixing this bug.

BTW, sorry for answering late - I was busy during the last couple of days.
